autoload/nvimdev.vim

```diff
@@ -56,15 +56,18 @@ function! nvimdev#init(path) abort
   endif
 
   " Use with :Neomake! make
+  " make will try to build  both dependancies and nvim
+  " Ignore entering .deps directory not to mess the quickfix-dirstack
+  let s:efm_backup=&efm
```

Comment:
This can just be a local variable without the `s:`.  Also, could you use the long form `errorformat`.  I don't like using abbreviations in plugins.